### DISCUSSION TITLE
Welcome to feed-master Discussions!

### DISCUSSION THREAD

**Discussion by umputun** [umputun/feed-master#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sasha-Ink** [comment]
Good Day! Thank you for your work!! I especially mean Feed Master (for YouTube feeds)..! It is very useful for people in Kremleputiya - because even VPN is often blocked by KGB here...(( God bless you!!
